src/assignment/components/AssignmentAdminFormEditable.tsx

```diff
@@ -0,0 +1,203 @@
+	};
+
+	const handleQualityLabelChange = (labels: TagInfo[]) => {
+		const newQualityLabels = labels.map(
+			(label) =>
+				({
+					assignment_id: assignment.id,
+					label: label.value,
+				} as Avo.Assignment.QualityLabel)
+		);
+
+		(setValue as any)('quality_labels', newQualityLabels, {
+			shouldDirty: true,
+			shouldTouch: true,
+		});
+		setAssignment((prev) => ({
+			...prev,
+			quality_labels: newQualityLabels,
+			blocks: (prev as Avo.Assignment.Assignment)?.blocks || [],
+		}));
+	};
+
+	const owner: PickerItem | undefined = assignment.profile
+		? {
+				label: `${assignment.profile.user.first_name} ${assignment.profile.user.last_name} (${assignment.profile.user.mail})`,
+				type: 'PROFILE',
+				value: assignment.profile.id,
+		  }
+		: undefined;
+
+	const handleOwnerChange = (newOwner: PickerItem | null) => {
+		if (isNil(newOwner)) {
+			return;
+		}
+		(setValue as any)('owner_profile_id', newOwner, {
+			shouldDirty: true,
+			shouldTouch: true,
+		});
+		setAssignment((prev) => ({
+			...prev,
+			owner_profile_id: newOwner.value,
+			blocks: (prev as Avo.Assignment.Assignment)?.blocks || [],
+		}));
+	};
+
+	return (
+		<Container mode="vertical">
+			<Container mode="horizontal">
+				<Form>
+					<Spacer margin="bottom">
+						<Grid>
+							<Column size="3-7">
+								<FormGroup label={tText('Laatst bewerkt door')}>
+									<TextInput
+										disabled
+										value={
+											getFullName(assignment.updated_by, true, false) || '-'
+										}
+									/>
+								</FormGroup>
+
+								<FormGroup label={tText('Aangepast op')}>
+									<TextInput
+										disabled
+										value={
+											assignment.updated_at
+												? formatTimestamp(assignment.updated_at)
+												: '-'
+										}
+									/>
+								</FormGroup>
+
+								<FormGroup label={tText("Briefing's")}>
+									<TextArea
+										height="auto"
+										value={assignment.briefing_id || undefined}
+										onChange={(value: string) => {
+											{
+												(setValue as any)('briefing_id', value, {
+													shouldDirty: true,
+													shouldTouch: true,
+												});
+												setAssignment((prev) => ({
+													...prev,
+													briefing_id: value,
+													blocks:
+														(prev as Avo.Assignment.Assignment)
+															?.blocks || [],
+												}));
+											}
+										}}
+									/>
+								</FormGroup>
+
+								{PermissionService.hasPerm(
+									user,
+									PermissionName.EDIT_ASSIGNMENT_QUALITY_LABELS
+								) && (
+									<FormGroup label={tText('Kwaliteitslabels')}>
+										{!!qualityLabels && (
+											<TagsInput
+												isLoading={isLoading}
+												options={
+													transformQualityLabelsToTagInfo(
+														qualityLabels
+													) || []
+												}
+												value={getAssignmentLabels()}
+												onChange={handleQualityLabelChange}
+											/>
```

Comment:
i prefer to keep a naming convention for these dropdowns:
* getSelectedQualityLabels() for the ones the user selected or the ones that are selected for the current assignment
and
* allQualityLabels for the full list of options that can be selected

It makes it easier to identify which is which if we keep this naming convention